doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -300,6 +301,20 @@ This section provides detailed lists of changes by :ref:`library <libraries>`.
 
   * :ref:`nrf_security` library, relocated from the sdk-nrfxlib repository to the :file:`subsys/nrf_security` directory.
 
+* :ref:`lib_hw_unique_key` library:
+
+  * Updated:
+
+    * :c:func:`hw_unique_key_write`, :c:func:`hw_unique_key_write_random` and :c:func:`hw_unique_key_load_kdr` functions to return an error code and not panic on error.
+    * :c:func:`hw_unique_key_derive_key` function to always return an error code from the library-defined codes and not forward error codes from underlying libraries.
+
+* :ref:`lib_identity_key`
```

Comment:
Amazingly enough, missed this in the first round :)
```suggestion
* :ref:`lib_identity_key` library:
```

Reply:
Done